From d6eae5c8f09d7dde6d1ad78c34c8dfb9ff3c31f4 Mon Sep 17 00:00:00 2001
From: Hideyuki Yamashita <yamashita.hideyuki@po.ntt-tx.co.jp>
Date: Mon, 5 Nov 2018 22:17:12 +0900
Subject: [PATCH 10/10] docs: refactor primary commands section

Misc update to adjust the length of underline of section header.

Signed-off-by: Hideyuki Yamashita <yamashita.hideyuki@po.ntt-tx.co.jp>
Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/commands/primary.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/guides/commands/primary.rst b/docs/guides/commands/primary.rst
index e49f990..a156966 100644
--- a/docs/guides/commands/primary.rst
+++ b/docs/guides/commands/primary.rst
@@ -2,7 +2,7 @@
     Copyright(c) 2010-2014 Intel Corporation
 
 Primary Commands
-====================
+================
 
 Primary process is managed with ``pri`` command.
 
-- 
2.7.4

